source/developers-guide/plugin-configuration/index.md

```diff
@@ -20,25 +20,36 @@ the configuration options for plugins. Shopware delivers a big amount of helper
 configuration fields in backend.
 
 ## Creating forms
-To create a simple plugin configuration we can use the function `Form()`
+Backend plugin configuration can be extended by Resources/config.xml file.
 
-```
-$form = $this->Form();
-```
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/shopware/5.2/engine/Shopware/Components/Plugin/schema/config.xsd">
+    <elements>
 
-Now we can use this instance of `Shopware\Models\Config\Form` in `$form` to add elements to it via `setElement().
-```
-$form->setElement(type : String, name : String, [options : array | null= null]);
+    </elements>
+</config>
 ```
+
+Now add elements in the elements tag.
+
 A textfield would be defined as followed:
 
-```
-public function createConfiguration()
-{
-    $form = $this->Form();
- 
-    $form->setElement('text', 'simpleTextField');
-}
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/shopware/5.2/engine/Shopware/Components/Plugin/schema/config.xsd">
+    <elements>
+        <element required="true" type="text" scope="locale">
+            <name>simpleTextField</name>
+            <label lang="de">Text</label>
+            <label lang="en">Text</label>
+            <value>Vorauswahl</value>
+            <description>Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut.</description>
```

Comment:
please also translate the description, to make clear that this is possible

Reply:
d